src/main/java/org/broadinstitute/hellbender/utils/runtime/ProcessSettings.java

```diff
@@ -48,6 +49,10 @@ public ProcessSettings(String[] command, boolean redirectErrorStream, File direc
         return command;
     }
 
+    public String getCommandString() {
+        return StringUtils.join(command, " ");
```

Comment:
why not use String.join?

Reply:
command is a String[], which matches the utils signature exactly.

Reply:
oh, good point